marklogic-data-hub-central/ui/src/pages/Curate.tsx

```diff
@@ -100,26 +101,32 @@ const Curate: React.FC = () => {
     }
   };
 
-
   return (
     <div className={styles.curateContainer}>
-      <div className={styles.intro}>
-        <p>{tiles.curate.intro}</p>
-      </div>
-      <EntityTiles
-        flows={flows}
-        canReadMatchMerge={canReadMatchMerge}
-        canWriteMatchMerge={canWriteMatchMerge}
-        canWriteMapping={canWriteMapping}
-        canReadMapping={canReadMapping}
-        canReadCustom={canReadCustom}
-        canWriteCustom={false}
-        entityModels={entityModels}
-        getEntityModels={getEntityModels}
-        canWriteFlow={canWriteFlow}
-        addStepToFlow={addStepToFlow}
-        addStepToNew={addStepToNew}
-      />
+      {
+        canReadMapping || canReadMatchMerge || canReadCustom ?
```

Comment:
I recommend abstracting this into "canCurate", "canRun", etc properties that are based on these authorities returned by the middle tier. We don't want the middle tier to determine what "canCurate" means because that would mean it needs knowledge of what's shown in the Curate tile.

Reply:
I'm not certain what you mean by this.  These booleans are already based on authorities given by authorityService, similar to how Run or Load tiles determine their read permission.

Reply:
Discussed and implemented